### PR TITLE
Added detailed description of proxy configuration

### PR DESCRIPTION
**Pull request by gkreyssing**
## Describe your changes
Added detailed description of proxy configuration and solved some spelling mistakes

### CODE REVIEW

File: content/en/docs/tutorials/quickstart/behind_proxy.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+# Environment Variables
+
+It is required to set the following environment variables:
+
+- `HTTP_PROXY` - proxy server, e.g. `http://localhost:3128`
+- `HTTPS_PROXY` - secure proxy server, e.g. `http://localhost:3128`
+- `DEVCONTAINER_PROXY` - Enables proxy configuration for the devContainer. Please use `.Proxy` as value and don't forget (dot).
+
+If you are running on Unix you have to define in addition the environment variable `DEVCONTAINER_PROXY_HOST` to define the internal docker host `172.17.0.1`.
```
Comment (rom3be) [inline content/en/docs/tutorials/quickstart/behind_proxy.md:41]: Can you add here title 
## For unix users: 
- If you are ...
- In additional ...

File: content/en/docs/tutorials/quickstart/behind_proxy.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+{{< /tabpane >}}
+
+# Visual Studio Code
+
+A template configuration using proxy settings is provided by our template repository with `.devcontainer/Dockerfile.Proxy`. By setting the environment variable `DEVCONTAINER_PROXY` to `.Proxy` the file
+`.devcontainer/Dockerfile.Proxy` will be used instead of `.devcontainer/Dockerfile`.
+
+# Troubleshooting
+
+If you experience issues during initial DevContainer build, clean all images and volumes otherwise cache might be used:
```
Comment (rom3be) [inline content/en/docs/tutorials/quickstart/behind_proxy.md:81]: Add sub-title:
## Case 1:

File: content/en/docs/tutorials/quickstart/behind_proxy.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+`.devcontainer/Dockerfile.Proxy` will be used instead of `.devcontainer/Dockerfile`.
+
+# Troubleshooting
+
+If you experience issues during initial DevContainer build, clean all images and volumes otherwise cache might be used: 
+
+   - Open Docker Desktop 
+   - From `Troubleshooting` choose `Clean / Purge data`
+
+Proxy settings in `.devcontainer/Dockerfile.Proxy` will be overridden by `~/.docker/config.json`, which can cause problems. In case the DevContainer is still not working, check if the `~/.docker/config.json` contains the following content:
```
Comment (rom3be) [inline content/en/docs/tutorials/quickstart/behind_proxy.md:86]: Add substitle: 

## Case 2:

**Review by mml5bg — APPROVED** [review]

File: content/en/docs/tutorials/quickstart/behind_proxy.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+   ```
+   {{% /tab %}}
+   {{% tab header="Linux" %}}
+   ```json
+   {
+    "proxies":{
+         "default":{
+            "httpProxy":"http://172.17.0.1:3128",
+            "httpsProxy":"http://172.17.0.1:3128",
+            "noProxy":"localhost,127.0.0.1,0.0.0.0,10.0.0.0/8,192.168.122.0/24,cattle-system.svc,.svc,.cluster.local"
```
Comment (rom3be) [inline content/en/docs/tutorials/quickstart/behind_proxy.md:109]: I think that noProxy could be same as for windows: 
"noProxy":"host.docker.internal,localhost,127.0.0.1"

**Review by rom3be — APPROVED** [review]

**Pull request merged**